buildenv

```diff
@@ -42,5 +36,5 @@ zopen_get_version()
 {
   # Modify to echo the version of your tool/library
   # Rather than hardcoding the version, obtain the version by running the tool/library
-  echo "1.0.0"
+  echo "2.21"
```

Comment:
Maybe you can put 2.21 into a global variable or extract the version from the which command

Reply:
I have modified this, thank you.